Scenarios/ITU-R_M2412-0_5BA.m

```diff
@@ -0,0 +1,50 @@
+clear all 
+
+% Disable cast to struct warnings
+w = warning ('off','all');
```

Comment:
This should not be necessary anymore. I think all the casting warnings are fixed no?

Reply:
They are, this is removed.